library/SimplePie.php

```diff
@@ -650,6 +650,13 @@ class SimplePie
 	 */
 	public $strip_htmltags = array('base', 'blink', 'body', 'doctype', 'embed', 'font', 'form', 'frame', 'frameset', 'html', 'iframe', 'input', 'marquee', 'meta', 'noscript', 'object', 'param', 'script', 'style');
 
+	/**
+	 * @var array Stores the default tags to be stripped by rename_attributes().
```

Comment:
If I am reading it correctly, the change does not seem to be about tags so the comment and variable names should be updated.

Reply:
fixed the comment